client/src/pages/UserLogin/UserLogin.js

```diff
@@ -40,35 +40,22 @@ const UserLogin = () => {
     }
   };
 
-  const handleSubmit = async (e) => {
-    e.preventDefault();
-    const username = e?.target[0]?.value;
-    const password = e?.target[1]?.value;
+  const handleSubmit = (username, password) => {
     if (username && password) {
       if (logIn) {
         return handleLogIn(username, password);
       }
       handleSignUp(username, password);
     }
   };
-
   return (
-    <div>
-      <h1>PET NAMES</h1>
-      <p>
-        Log In to access site{' '}
-        <span>
-          <button onClick={handleChange}>Log in</button>
-        </span>
-      </p>
-      <p>
-        Don't have an account?{' '}
-        <span>
-          <button onClick={handleChange}>Sign up</button>
-        </span>
-      </p>
-      <LogInContainer logIn={logIn} handleSubmit={handleSubmit} />
-    </div>
+    <>
+      <LogInContainer
+        logIn={logIn}
```

Comment:
mas nice guro nga ang variable name is `isLogginIn` para mas explicit siya

Reply:
if boolean ang variable usually naa jud na nga mga prefix, `is` `has` etc.

Reply:
sige sige